### DISCUSSION TITLE
Apollo Save Tool v1.8.4

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#115] [General]
### Added

* Hex Editor for save-data files
* Improved internal Web Server (Online DB support)
* Custom offline Account-ID activation (On-screen Keyboard)
* User-defined Online DB URL (`Settings`)

### Fixed

* Fix Final Fantasy XIII save-game decryption bug

### Misc

* Updated Apollo patch engine v0.4.1
  * Skip search if the pattern was not found
  * Improve code types 9, B, D
  * Fixed Final Fantasy XIII encryption bug on PS3
  * Add value subtraction support (BSD)
* Custom checksum: Dai-3-Ji Super Robot Taisen Z

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v1.8.4'>Apollo Save Tool v1.8.4</a>.</em>

**Comment by MichaelX17** [comment]
RIP... I really hope you gave them a very good life, man

**Comment by Phoenixx1202** [comment]
How do I fix this problem?

![Erro Libxmp-lite](https://github.com/bucanero/apollo-ps3/assets/26288409/2abcac60-46e9-41e0-a116-e27bb6eb77c5)
